### PR TITLE
Add mod_wsgi build script

### PR DESCRIPTION
**Pull request by vedang-wartikar**
Added build script for mod_wsgi package. Tested with tox, all test cases passing

### CODE REVIEW

**Review by chaitanyarahalkar — APPROVED** [review]

**Comment by seth-priya** [comment]
@Ryan-Dmello please review

File: m/mod_wsgi/mod_wsgi_CENTOS8.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export PIP=pip3
+export REPO=https://github.com/GrahamDumpleton/mod_wsgi
+
+yum update -y
+
+# install python
+yum install ${PYTHON} ${PYTHON}-devel ${PYTHON}-pip -y
+
+# install mod_wsgi and related dependencies
+yum install ${PYTHON}-mod_wsgi httpd httpd-devel gcc redhat-rpm-config -y
```
Comment (Ryan-Dmello) [inline m/mod_wsgi/mod_wsgi_CENTOS8.sh:29]: I dont think we need **redhat-rpm-config** package. Please check and remove

File: m/mod_wsgi/mod_wsgi_CENTOS8.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : mod_wsgi
+# Version       : 4.7.1
+# Source repo   : https://github.com/GrahamDumpleton/mod_wsgi
+# Tested on     : RHEL 7.6, RHEL 7.7
```
Comment (Ryan-Dmello) [inline m/mod_wsgi/mod_wsgi_CENTOS8.sh:6]: Please update naming convention of file, based on the above line

File: m/mod_wsgi/mod_wsgi_CENTOS8.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# install mod_wsgi and related dependencies
+yum install ${PYTHON}-mod_wsgi httpd httpd-devel gcc redhat-rpm-config -y
+
+# install git repo for testing
+yum install git -y
+${PIP} install tox
+git clone https://github.com/GrahamDumpleton/mod_wsgi
+cd mod_wsgi
+tox -e py36
```
Comment (Ryan-Dmello) [inline m/mod_wsgi/mod_wsgi_CENTOS8.sh:36]: As per line 4, script is targeted for version 4.7.1. Please update accordingly.

**Review by Ryan-Dmello — CHANGES_REQUESTED** [review]

File: m/mod_wsgi/mod_wsgi_CENTOS8.sh (hunk 1 above)
Comment (vedang-wartikar) [reply m/mod_wsgi/mod_wsgi_CENTOS8.sh:29]: The build is failing, if not installed, with error message
```
  gcc: error: /usr/lib/rpm/redhat/redhat-hardened-cc1: No such file or directory
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for mod-wsgi
```

File: m/mod_wsgi/mod_wsgi_RHEL7.6.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+export REPO=https://github.com/GrahamDumpleton/mod_wsgi
+export VERSION=4.7.1
+
+yum update -y
+
+# install python
+yum install ${PYTHON} ${PYTHON}-devel ${PYTHON}-pip -y
+
+# install mod_wsgi and related dependencies
+yum install ${PYTHON}-mod_wsgi httpd httpd-devel gcc redhat-rpm-config -y
```
Comment (Ryan-Dmello) [inline m/mod_wsgi/mod_wsgi_RHEL7.6.sh:30]: If package is available via yum package manager, no need for build script

**Review by Ryan-Dmello — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
do we need this PR then @Ryan-Dmello @vedang-wartikar ?

**Comment by Ryan-Dmello** [comment]
The Build script is not required, as the package installation is done via yum installer. This PR is invalid

**Pull request closed without merging**